<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemWidget</class>
 <widget class="QWidget" name="ItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,3,0">
   <item>
    <widget class="QLabel" name="imageLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Info</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset theme="system-run"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rewatchButton">
       <property name="text">
        <string>Rewatch</string>
       </property>
       <property name="icon">
        <iconset theme="edit-undo"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset theme="media-playback-start"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="text">
        <string>Finish</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="incrementButton">
       <property name="text">
        <string>Increment</string>
       </property>
       <property name="icon">
        <iconset theme="contact-new"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset theme="edit-delete"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset theme="document-properties"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
